### PR TITLE
Add tooltip

### PR DESCRIPTION
**Pull request by ej9x**

### CODE REVIEW

File: src/components/Tooltip/Tooltip.theme.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const name = 'tooltip';
+
+const tooltipMessageTheme = createComponentTheme(`${name}Message`, ({ COLORS, SIZES }: *) => ({
+  root: {
+    padding: '12px',
+    backgroundColor: COLORS.BLACK,
+    color: COLORS.WHITE,
+    fontSize: SIZES.OVERLINE_2,
+    borderRadius: SIZES.MAIN_BORDER_RADIUS,
+    boxSshadow: '2px 0 10px 0 rgba(0, 0, 0, 0.16)',
```
Comment (zouxuoz) [inline src/components/Tooltip/Tooltip.theme.js]: @ej9x `boxShadow`

File: src/components/Tooltip/Tooltip.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    openTooltip =() => this.setState({ isOpen: true });
+    closeTooltip = () => this.setState({ isOpen: false });
+    toggleTooltip = () => this.setState({ isOpen: !this.state.isOpen });
+
+    handleClickOutside = (event: *) => {
+      this.closeTooltip();
+      event.stopPropagation();
+    };
+
+    renderTooltipoMessage = () => {
```
Comment (zouxuoz) [inline src/components/Tooltip/Tooltip.js]: @ej9x `Tooltipo`?

File: src/components/Tooltip/Tooltip.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      super(props);
+
+      this.state = {
+        isOpen: props.defaultOpen || false,
+      };
+    }
+
+    openTooltip =() => this.setState({ isOpen: true });
+    closeTooltip = () => this.setState({ isOpen: false });
+    toggleTooltip = () => this.setState({ isOpen: !this.state.isOpen });
```
Comment (vorobeez) [inline src/components/Tooltip/Tooltip.js]: @ej9x Please, rewrite it as async setState.
```
this.setState(({ isOpen }) => ({
  isOpen: !isOpen,
}));
```

File: src/components/Tooltip/Tooltip.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+    handleClickOutside = (event: *) => {
```
Comment (vorobeez) [inline src/components/Tooltip/Tooltip.js]: @ej9x Is this function used in the code?

Comment (ej9x) [reply src/components/Tooltip/Tooltip.js]: yes. part of usage react-onclickoutside

**Pull request merged**